pkg/loki/labels.go

```diff
@@ -0,0 +1,36 @@
+package loki
+
+import (
+	"strconv"
+
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+	"github.com/netobserv/network-observability-operator/controllers/ebpf"
+	"github.com/netobserv/network-observability-operator/pkg/helper"
+)
+
+func GetLokiLabels(desired *flowslatest.FlowCollectorSpec) []string {
+	indexFields := constants.LokiIndexFields
+
+	if desired.Processor.LogTypes != nil && *desired.Processor.LogTypes != flowslatest.LogTypeFlows {
+		indexFields = append(indexFields, constants.LokiConnectionIndexFields...)
+	}
+
+	if desired.Processor.MultiClusterDeployment != nil && *desired.Processor.MultiClusterDeployment {
+		indexFields = append(indexFields, constants.ClusterNameLabelName)
+	}
+
+	if helper.UseEBPF(desired) {
```

Comment:
so you are using justMark bool as indication if dupMerge or not ? why not check dupMerge directly ?

Reply:
/!\ This file only manages loki labels ! 

For now, I only add / remove `FlowDirection`  and `Duplicate` labels according to `justMark` flag: 
```
var LokiDeduperMarkIndexFields = []string{"FlowDirection", "Duplicate"}
```

For `merge` flag, we don't need label on related arrays since there is no deduplication on plugin side. That means queries are running straight forward without making any fancy filtering on these :sunglasses: